### PR TITLE
feat(info): add btn to serebii

### PR DESCRIPTION
**Pull request by ami**
ヾ(０∀０*★)ﾟ*･.｡

### CODE REVIEW

File: app/views/info.html [hunk 1]
```diff
@@ -30,7 +30,8 @@ <h1 [innerHTML]="active.html_name"></h1>
 
   <evolution-family *ngIf="active.evolution_family && !loading" class="info-evolutions" [family]="active.evolution_family" (activeChange)="active = $event"></evolution-family>
 
-  <a class="info-footer" angulartics2On="click" angularticsEvent="open Bulbapedia link" angularticsCategory="Info" angularticsProperties="new Object({ label: '{{active.name}}' })" [href]="active.bulbapedia_url" target="_blank">
-    <div>View on Bulbapedia <i class="fa fa-long-arrow-right"></i></div>
-  </a>
+  <div class="info-footer">
+    <a angulartics2On="click" angularticsEvent="open Bulbapedia link" angularticsCategory="Info" angularticsProperties="new Object({ label: '{{active.name}}' })" [href]="active.bulbapedia_url" target="_blank">Bulbapedia <i class="fa fa-long-arrow-right"></i></a>
+    <a angulartics2On="click" angularticsEvent="open Bulbapedia link" angularticsCategory="Info" angularticsProperties="new Object({ label: '{{active.name}}' })" [href]="active.serebii_url" target="_blank">Serebii <i class="fa fa-long-arrow-right"></i></a>
```
Comment (robinjoseph08) [inline app/views/info.html]: `angularticsEvent="open Serebii link"`

File: app/styles/tracker.scss [hunk 2]
```diff
@@ -199,41 +199,57 @@ info {
...
+      font-family: $font-secondary;
+      font-size: 11px;
+      font-weight: 700;
+      text-transform: uppercase;
+      letter-spacing: 1px;
+      text-decoration: none;
+      text-align: center;
+
+      &:last-child {
+        background-color: $brand-secondary-dark;
```
Comment (robinjoseph08) [inline app/styles/tracker.scss]: is it weird that the two buttons are different colors? when it first loaded, i kinda thought that the serebii link was already "selected" or something. do you not think so?

Comment (ami) [inline app/styles/tracker.scss]: yea idk why i did this. the prestige got me all confused. updating to a new design soon!

Comment (robinjoseph08) [inline app/styles/tracker.scss]: lol it'll do that to ya

**Comment by ami** [comment]
@robinjoseph08 updated!

**Comment by robinjoseph08** [comment]
lgtm!

**Pull request merged**